src/sentry/sentry_metrics/indexer/models.py

```diff
@@ -50,3 +58,146 @@ class Meta:
         constraints = [
             models.UniqueConstraint(fields=["string", "organization_id"], name="unique_org_string"),
         ]
+
+    @classmethod
+    def get_many_ids(cls, keys) -> Mapping[str, str]:
+        # keys => "organization_id:string"
+        final_results = {}
+        cache_results = indexer_cache.get_many(keys, field_type=str)
+        # {"key" => result}
+        db_look_up_keys = set()
+        for key in keys:
+            cache_key = indexer_cache.make_cache_key(key, field_type=str)
+            result = cache_results.get(cache_key)
```

Comment:
Why isn't this hidden inside `indexer_cache`? 
If you pass a list of `org_id:string` to `indexer_cache` you would expect back something like
```
{
    `org_id:string`: string,
    .....
}
```
And not the hashes of the keys that are the memcached keys. Those hashes seem a StringIndexerCache implementation detail that should not get out of it.